### ISSUE TITLE
Bug report: Booking notification

### ISSUE THREAD

**Issue by dashamstyr** [daisieh/lmc_carshare#2]
I booked the Toyota Prius, which did not require confirmation but the notification email read:

Your vehicle Blue Toyota Prius AL675T has been requested for
2020-12-31T20:00:00-08:00 to 2020-12-31T21:00:00-08:00.

**Comment by daisieh** [comment]
Yeah, I had that all backwards. I think it's fixed now. Please verify?

**Comment by dashamstyr** [comment]
Bug appears to remain  Just tried to book Toyota prius:

1) Pop-up said waiting for confirmation of booking (although car does not require confirmation)
2) No email received (maybe still waiting to be sent?)
3) booking shows up under requests tab

Repeated this twice with same result
